src/test/java/fi/metatavu/metaform/test/functional/AuditLogEntryTestsIT.java

```diff
@@ -0,0 +1,129 @@
+package fi.metatavu.metaform.test.functional;
+
+import feign.FeignException;
+import fi.metatavu.metaform.client.api.*;
+import fi.metatavu.metaform.client.model.*;
+import fi.metatavu.metaform.server.rest.ReplyMode;
+import fi.metatavu.metaform.server.rest.translate.MetaformTranslator;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.*;
+
+
+import static io.restassured.RestAssured.given;
+import static org.junit.Assert.*;
+
+public class AuditLogEntryTestsIT extends AbstractIntegrationTest {
+
+
+    /**
+     * test creation of audit logs for reply creation, viewing and deleting
+     * @throws IOException
+     * @throws URISyntaxException
+     */
+    @Test
+    public void createReplyAuditEntry() throws IOException{
```

Comment:
Prefix tests with Test
```suggestion
    public void createReplyAuditEntryTest() throws IOException{
```